templates/styles/build_passed.md.erb

```diff
@@ -1 +1,3 @@
 :smiley: Your submission passed all our automated tests.
+
```

Comment:
This is used for locales and styles, right? I'd like to add something to the extent of: 

> Below are some sample citations generated by your proposed changes to citation styles

But that won't work for locales (where we don't produce output, right?). Can we differentiate? Any other ideas? This doesn't have to be perfect, just shouldn't be confusing.

Reply:
No, `styles` and `locales` have their own build templates, each can be edited separately.

Reply:
I could have figured that out from the path...